source/kraken/make_disruption_from_chaos.cpp

```diff
@@ -36,6 +36,26 @@ namespace bt = boost::posix_time;
 
 namespace navitia {
 
+static void update_property(nt::disruption::Property& property, const chaos::Property& chaos_property) {
+    property.key = chaos_property.key();
+    property.type = chaos_property.type();
+    property.value = chaos_property.value();
+}
+
+static boost::shared_ptr<nt::disruption::Property>
+make_property(const chaos::Property& chaos_property, nt::disruption::DisruptionHolder& holder) {
+    auto& weak_property = holder.properties[chaos_property.key() + chaos_property.type() + chaos_property.value()];
```

Comment:
please make a function to abstract the key's construction (`make_property_key(...)`)

Reply:
Done.